src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -607,4 +644,9 @@ private DBOptions createOptions(String logPath, String configFile) throws IOExce
 
         return options;
     }
+    
+    @Override
+    public long getPersistanceSize() {
```

Comment:
This is a little thing that annoys me (found out just now)...
You have a Typo  -> Persistence

Reply:
uhhhhg every time i type that word wrong :p